chemprop/cli/train.py

```diff
@@ -382,7 +382,7 @@ def add_train_args(parser: ArgumentParser) -> ArgumentParser:
     train_args.add_argument("--max-lr", type=float, default=1e-3, help="Maximum learning rate.")
     train_args.add_argument("--final-lr", type=float, default=1e-4, help="Final learning rate.")
     train_args.add_argument(
-        "--epochs", type=int, default=50, help="the number of epochs to train over"
+        "--epochs", type=int, default=50, help="The number of epochs to train over."
```

Comment:
Leaving incomplete sentences as lowercase and not punctuated was our plan for formatting which would undo this change. What do you think of that? Happy to hear your editorial thoughts on our style.

Reply:
We can go ahead and do that, I don't have a strong preference. This change was to just make it consistent with the helpstrings for other functions. The question is - do we leave this in with this PR, or undo it?

Reply:
Looking at other changes in this PR, I see there are many that remove the period but keep the capital of incomplete sentences (such as `"Aggregation mode to use during graph predictor"`). Can you remove the period here and in the other instances of incomplete sentences (such as `"Final learning rate."`), but keep the capitals? I also see Anna changed `"the weight to apply ..."` to `"Weight to apply ..."`. To be consistent here should probably be `"Number of epochs ..."`, but I don't have much of an opinion about that. (side note that `"Weight to apply ..."` should probably be `"Weights to apply ..."`, but I have no opinion if that should be changed in this PR or another one.)

Reply:
Thanks for the suggestion, that's a good catch. I addressed this in the latest commit, let me know what you think. I ended up making those wording changes you suggest in the 2nd half of your comment as well because I noticed some other spots where the consistency could also be improved, so I thought we'd just address them all here.